tests/test_models/test_backbones/test_sr_backbones/test_basicvsr_plusplus.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import torch
+
+from mmedit.models.backbones.sr_backbones.basicvsr_pp import BasicVSRPlusPlus
+
+
+def test_basicvsr_plusplus():
+    """Test BasicVSR++."""
+
+    # gpu (DCN can only be tested in GPUs)
```

Comment:
Can DCN be turned off, and be tested on cpu?

Reply:
Yes we can, but we need to create another argument to specify whether to skip the alignment.

Reply:
Since this is one of the core parts of BasicVSR++, I am not sure whether adding this option is meaningful or not.

Reply:
I just want to test it haha. It does not have to do with what basicvsr thinks

Reply:
Or can you implement a cpu version of DCN?

Reply:
> Or can you implement a cpu version of DCN?

Not sure if I can do it in a short time haha. One workaround for now is to add the argument to skip alignment, and set it to False.

Reply:
Or you can check the coverage report to see if more parts could be tested. 80% is a healthy line